vm.go

```diff
@@ -584,11 +582,6 @@ func (vm *vm) run() {
 		} else {
 			vm.prg.code[vm.pc].exec(vm)
 		}
-		ticks++
```

Comment:
q: how does this `ticks` differ from the `Runtime.ticks` field? it seems like this `ticks` might've been doing the same thing as `self.ticks` that we consume in the `Runtime.waltOneTick` helper?

Reply:
If so, i think I feel better amount removing this local `ticks` var

Reply:
Yeah so `ticks` was introduced [here](https://github.com/Calvinnix/goja/commit/f95411d186f2183429261010a860c604371ac6a7) by the maintainer. It provides a manner to yield back to the CPU.

Our self.ticks is used for rate limiting (basing off of the git commits), so I guess its fairly similar, I just still need to confirm that the waitN logic yields to the CPU like I'm assuming it will.

Reply:
hmm that mostly sounds good to me though there is the risk that this behavior might change over time in different versions of go